apps/aecore/src/aec_peer_connection.erl

```diff
@@ -1091,22 +1091,24 @@ send_response(S, Type, Response) ->
     Msg = aec_peer_messages:serialize_response(Type, Response),
     send_msg(S, response, Msg).
 
-%% If the message is more than 65533 bytes it won't fit in a single Noise
-%% message and we need to fragment it.
+%% If the message is more than 65535 - 2 - 16 bytes (2 bytes for the length and
+%% 16 bytes for the encryption MAC) it won't fit in a single Noise message and
+%% we need to fragment it.
 %%
 %% Fragments have the format <<?MSG_FRAGMENT:16, N:16, M:16,
-%% PayLoad:65529/binary>> (where the last fragment has a smaller Payload),
-%% saying that this is fragment N out of M fragments.
+%% PayLoad:65529/binary>> (where the last fragment has a smaller Payload, and
```

Comment:
Shall not each fragment consider `- 2 - 16`?

Reply:
> where 16 bytes of the payload is the encryption MAC

I see 16 considered. Not sure about 2.

Reply:
Yes you are right, I think the ` - 2` should really go into `MAX_PACKET_SIZE` that would make the description better and the code easier to follow?!

Reply:
Yes.

Reply:
I see 65529 now 65527. Ok.